vuu-ui/packages/vuu-shell/src/layout-management/LayoutList.tsx

```diff
@@ -0,0 +1,51 @@
+
+import { List } from '@finos/vuu-ui-controls';
+import './LayoutList.css'
+
+export type LayoutMetadata = {
+    name: string,
+    group: string,
+    screenshot: string,
+    user: string,
+    date: string
+}
+
+export const LayoutsList = (props: { layouts: LayoutMetadata[] }) => {
+
+    const { layouts } = props;
+
+    const layoutsByGroup = layouts.reduce((acc: { [groupName: string]: LayoutMetadata[] }, cur) => acc[cur.group] ? { ...acc, [cur.group]: [...acc[cur.group], cur] } : { ...acc, [cur.group]: [cur] }, {})
+
+    return (
+        <>
+            <div className='vuuLayoutList-header'>My Layouts</div>
+            <List<[string, LayoutMetadata[]]>
+                height='fit-content'
+                source={Object.entries(layoutsByGroup)}
+                ListItem={({ item }) => <>
+                    <div className="vuuLayoutList-groupName">{item?.[0]}</div>
+                    <List<LayoutMetadata>
+                        height='fit-content'
+                        source={item?.[1]}
+                        ListItem={({ item }) =>
```

Comment:
Am I right in thinking the nested arrow functions give us an "inner" and "outer" definition of `item`? It might be worth renaming one or both to avoid ambiguity.